sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -1653,18 +1653,21 @@
           {
             "name": "deviceId",
             "in": "path",
+            "description": "The Id of the device on whose module the direct method is to be invoked.",
```

Comment:
This sentence structure is a bit hard to follow. Maybe "The Id of the device on whose module the direct method will be invoked."